docs/data/sql_functions.yml

```diff
@@ -933,11 +933,13 @@ valueconstruction:
   - sql: |
       -- implicit constructor with parenthesis
       (value1 [, value2]*)
+      -- explicit ROW constructor 
```

Comment:
The rendered doc seems not respecting newlines, could you fix that?

Reply:
how do you see the rendered doc ? build locally ?